bundle/src/main/java/com/cognifide/cq/cqsm/core/scripts/ScriptContent.java

```diff
@@ -32,25 +32,25 @@
 @Getter
 public class ScriptContent {
 
-  public static final String CQSM_FILE = "cqsm:File";
+  public static final String CQSM_FILE = "apm:Script";

```

Comment:
What about CQSM in constant name?

Reply:
Whoa! I was certain that I did this, thanks!